sdk/src/transaction_context.rs

```diff
@@ -1040,8 +1040,8 @@ impl<'a> BorrowedAccount<'a> {
 
     /// Returns whether this account is executable (transaction wide)
     #[inline]
-    pub fn is_executable(&self, feature_set: &FeatureSet) -> bool {
-        is_builtin(&*self.account) || is_executable(&*self.account, feature_set)
+    pub fn is_executable(&self, _feature_set: &FeatureSet) -> bool {
+        is_builtin(&*self.account) || self.account.executable()
```

Comment:
It was only `self.account.executable()` before not `is_builtin(&*self.account) ||`.

Reply:
You're right, ` is_builtin(&*self.account) ||` should not be necessary - I left it in as belt and braces. 

I've update the PR and my dev box.

Reply:
There were two more occurrences of `is_builtin() ||` which I've removed.